[DOCS-12851] restructure Cloudcraft overlays section + CCM updates (#33471)
DataDog/documentation: content/en/datadog_cloudcraft/overlays/ccm.md

---
title: Cloud Cost Management
description: "Use the Cloud Cost Management overlay in Cloudcraft to view resource-level costs and discover savings opportunities directly on your architecture diagrams."
further_reading:
- link: "/datadog_cloudcraft/overlays/infrastructure/"
  tag: "Documentation"
  text: "Infrastructure overlay"
- link: "/datadog_cloudcraft/overlays/observability/"
  tag: "Documentation"
  text: "Observability overlay"
- link: "/datadog_cloudcraft/overlays/security/"
  tag: "Documentation"
  text: "Security overlay"
- link: "/cloud_cost_management/"
  tag: "Documentation"
  text: "Cloud Cost Management"
---

## Overview

The Cloud Cost overlay helps you visualize resource-level costs and identify savings opportunities within your AWS architecture diagrams. This overlay provides two views: **[Cost](#cost-view)** and **[Recommendations](#recommendations-view)**.

## Cost view

> The Cost view displays resource-level cloud costs for the last 30 days, allowing you to understand spending patterns across your architecture.

Reviewer: ```suggestion
The Cost view shows resource-level cloud costs from the past 30 days, allowing you to understand spending patterns across your architecture.
```
